### ISSUE TITLE
font-size definitions shouldn’t be in absolute units

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail.org#322]
There are a large number of `font-size` definitions set in `px` values across the project. This is problematic for users of browser-level text zooming. All those definitions should be replaced with equivalent `rem` values.

**Comment by Aarthy153** [comment]
Hi, I would like to work on this issue. Can I get assigned?

**Comment by zerolab** [comment]
@Aarthy153 we do not assign issues to non-core folk. Add a note that you are working on the issue, then open a PR

**Comment by kituuu** [comment]
@thibaudcolas @zerolab, can I work on it if the issue is still not fixed?

**Comment by thibaudcolas** [comment]
Anyone can work on anything, please read the comment right above yours.
